contract/contracts/Tips.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+contract Tips {
+    mapping(address => uint256) private addressToTips;
```

Comment:
`mapping(address => uint256) private addressToTips;` 
This mapping is storing a uint256 value for each address. So you should name it balance.